addons/web/controllers/webclient.py

```diff
@@ -148,3 +148,10 @@ def bundle(self, bundle_name, **bundle_params):
         } for tag, attrs, content in files]
 
         return request.make_json_response(data)
+
+    @http.route('/web/webclient/check_record_company', type='json', auth="user")
+    def check_record_company(self, model, res_id, company_ids):
+        company_dependant = hasattr(request.env[model], 'company_id')
```

Comment:
If you wanna check that a field is present on a model, we usually do this instead
```suggestion
        company_dependant = 'company_id' in request.env[model]
```

Reply:
done